src/modules/cmdpal/exts/Microsoft.CmdPal.Ext.Apps/Commands/OpenInConsoleCommand.cs

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Diagnostics;
+using System.Threading.Tasks;
+using Microsoft.CmdPal.Ext.Apps.Properties;
+using Microsoft.CommandPalette.Extensions.Toolkit;
+
+namespace Microsoft.CmdPal.Ext.Apps.Commands;
+
+// NOTE this is pretty close to what we'd put in the SDK
+internal sealed partial class OpenInConsoleCommand : InvokableCommand
+{
+    private readonly string _target;
+
+    public OpenInConsoleCommand(string target)
+    {
+        Name = Resources.open_path_in_console;
+        Icon = new IconInfo("\ue838");
```

Comment:
nit: we can probably save some memory by having a single static instance of this IconInfo for all `OpenInConsoleCommand`'s, but who knows if that's significant